app/views/animals/new.html.erb

```diff
@@ -2,7 +2,8 @@
 <hr>
 <h2>New Animal Form</h2>
 <div class="new_animal_form">
-  <%= form_with url: "/shelters/#{@facade.shelter.id}/animals/#{animal.id}", method: :post, data: {turbo: false} do |form| %>
+  <%# the form_with url may not be correct %>
+  <%= form_with url: "/shelters/#{@facade.shelter.id}/animals/#{@facade.animal.id}", method: :post, data: {turbo: false} do |form| %>
```

Comment:
it would be `"/animals/create"` right?? I'm blanking

Reply:
I believe you are correct!